app/schemas/models/level_component.coffee

```diff
@@ -34,6 +34,7 @@ PropertyDocumentationSchema = c.object {
   type: c.shortString(title: 'Type', description: 'Intended type of the property.')
   shortDescription:
     oneOf: [
+      {title: 'Short Description', type: 'string', description: 'Short Description of the property.', maxLength: 1000, format: 'markdown'}
```

Comment:
Why was this moved?

Reply:
Yep this was moved so that while adding short description from the level editor, by default the type `string` is selected (i.e. generic description for all languages) instead of language-specific description object. This is done for the ease of content creation because most of the time there will be generic short description, and can manually change it to language-specific type if required. (Same for the property below)